script/autoapi/engine.cpp.j2

```diff
@@ -0,0 +1,582 @@
+// Part of Measurement Kit <https://measurement-kit.github.io/>.
+// Measurement Kit is free software under the BSD license. See AUTHORS
+// and LICENSE for more information on the copying conditions.
```

Comment:
This is the template for `engine.cpp`. Is it readable? (I'll comment on specific choices in `engine.cpp`.)